qmd/wide-names.qmd

```diff
@@ -0,0 +1,132 @@
+
+## Protocol steps
+
+Like measures, protocol step wide-names record the `value` field. With protocols, the wide-name formula depends on whether the step uses a `measure` or a `method`. The formula if using a `method` is: `tableShortName_partTypeShortName_method_attribute`.
+
+The formula if using a `measure` is: `tableShortName_partTypeShortName_measure_unit_aggregation_index_attribute`, where in both instances, the table short name always specifies the `protocolSteps` table (`ps`), and the `partType` short name specifies whether this is a measure (`mes`) or a method (`met`). The attribute is always `value`.
+
+For example, the wide-name for the sequencing strategy protocol step (ie. a protocol step using a method) for a wastewater program would be `ps_met_seqStrat_value`. The protocol steps table short name indicates the long-format table (`ps`), the protocol step uses a method part type (`met`), it reports the sequencing strategy (`seqStrat`), and the column records the value (`value`).
+
+Similarly, the wide-name for a storage temperature (i.e. a protocol step using a measure), would be `ps_mes_temp_cel_sin_NR_value`. The protocol steps table short name indicates the long-format table (`ps`), the protocol step uses a measure part type (`mes`), it reports the temperature (`temp`) in degrees Celsius (`cel`), it is a single measure (`sin`, i.e. not aggregated), the index is not reported (`NR`), and the column records the value (`value`).
+
+## For combined long- and wide-formats
+
+Many environmental surveillance and research programs use combined long- and wide-format tables. To account for this, wide-names can contain a "see header" part for each piece of their formulas. These reference the header they point to, promoting data readability for machines and humans.
+
+For example, for a measure of the N1 allele of SARS-CoV-2, where the fraction analyzed in specified in a different column would use the wide-name `wat_sa_hFr_covN1_gcml_m_value`. Here the `hFr` indicates by its placement and the letters used that the fraction analyzed is specified elsewhere.
+
+Another example is for data is reported in long-format for one measure, but wide-format for all others. In this situation the fraction analyzed, the measure, the units, and the aggregation are all specified in other columns. Rather than simply reporting `value` as you would an attribute, you would use the wide name: `wat_sa_hFr_hMr_hUn_hAg_NR_value` to better link all the metadata pieces in other columns to this specific measurement.
+
+# Wide-name exceptions
+
+There is a diversity of wide-templates across labs. Sometimes these headers can combine two ODM attributes, or two or more methods or measures. This complicates mapping to the ODM, but unique wide-names can be generated for these as well.
+
+## Attribute naming exceptions
+
```

Comment:
I suggest separating the fomulas from the bulk of the text using triple backticks for readability